src/main.ts

```diff
@@ -25,7 +25,12 @@ async function bootstrap() {
   app.useGlobalPipes(new ValidationPipe());
 
   // add security middleware
-  app.use(helmet());
+  app.use(
+    helmet({
+      contentSecurityPolicy:
+        process.env.NODE_ENV === 'production' ? undefined : false,
```

Comment:
Why the need for a difference between dev and production here? 
And why is the contentSecurityPolicy `undefined` for production?

Reply:
Yeah, I'm going to change this for false, for all of env's, cause this is needed for graphql playground to be able to run.